### ISSUE TITLE
Single quote in field name crashes building search

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#791]
From @nllong, see #550 
if you map a field to a new name with a single quote it crashes the building search. I'm sure there are a few characters that will cause this to crash.

syntax error at or near "s"
LINE 1: ..."active" = true AND (LOWER((extra_data->>'Phylroy's Phun Phield...

**Comment by nicholasserra** [comment]
@nllong Can I get some steps to reproduce on this? Thanks!

**Comment by nllong** [comment]
To recreate:
* Import a data file
* In the mapping screen, change one of the names to something with a single quote (and probably other characters). 
* Map the data, then view the buildings. It should crash there, if not it would probably crash when you are looking at a specific building.

Let me know if you need more details.

**Comment by nicholasserra** [comment]
@nllong 

So I tried quotes and ticks on the mapping page, mapped, matched, viewed buildings with no issue. Anything I might be missing?

![screen shot 2016-03-21 at 4 27 02 pm](https://cloud.githubusercontent.com/assets/512288/13933097/e4c347da-ef81-11e5-90d7-e5ae36a0bfc6.png)

![screen shot 2016-03-21 at 4 27 28 pm](https://cloud.githubusercontent.com/assets/512288/13933100/e7d76a14-ef81-11e5-9c85-b7d87564fe5c.png)

**Comment by nllong** [comment]
that was the issue. Let me try again.

**Comment by nllong** [comment]
I tested this again locally and i didn't see the issue anymore. closing...

**Comment by RDmitchell** [comment]
am reopening to test on seedtest
